dnf/cli/commands/__init__.py

```diff
@@ -768,8 +768,9 @@ def __init__(self, cli):
             alias: subcmd for subcmd in subcmd_objs for alias in subcmd.aliases}
 
     def set_argparser(self, parser):
-        super(OptionParser, parser).add_argument('reponame', nargs=1,
-                                                 metavar=_('REPO'))
+        super(OptionParser, parser).add_argument(
+            'reponame', nargs=1, action=OptionParser._RepoCallbackEnable,
```

Comment:
@j-mracek can you show some example how it should be used? And functional test would be even better ;)